content/en/gitops.md

```diff
@@ -1,31 +1,25 @@
 ---
 title: GitOps
-status: Feedback Appreciated
+status: Completed
 category: Concept
-tags: ["", "", ""]
+tags: ["methodology"]
 ---
 
 ## What it is
 
-GitOps is a set of best practices based on [shared principles](https://opengitops.dev/), 
-applied to a workflow that depends on software agents that 
-enable automation to reconcile a declared system state or configuration in a git repository.
-These software agents and practices are used to execute a cohesive workflow that 
-leverages a source control system like Git as the “single source of truth” and 
-extends this practice to applications, infrastructure, and operational procedures.
+GitOps is a set of best practices based on [shared principles](https://github.com/open-gitops/documents/blob/release-v1.0.0/PRINCIPLES.md), applied to a workflow that depends on software agents to continuously and automatically reconcile an actual running system with its desired state declared in an immutably versioned state store like Git.
+These software agents and practices are used to execute a cohesive workflow and extends this practice to applications, infrastructure, and operational procedures.
 
 ## Problem it addresses
 
-Existing processes for infrastructure configuration management can face challenges 
-such as configuration drift, failed deployments, relying on a system's previous state for success, 
-missing documentation, or unknown development history.
+Existing processes for infrastructure configuration management – including scripting and tightly coupled CI/CD – are often error prone and can face challenges such as configuration drift, failed deployments, relying on a system's previous state for success, missing documentation, or unknown development history.
+Event driven CD does not address drift that may happen independently of CI.
+Giving external automation tools credentials to your production systems widens the attack footprint.
```

Comment:
When and why would you give credentials your automation tools? Seems like context is missing here.